client/src/App.tsx

```diff
@@ -30,6 +31,7 @@ function App(): JSX.Element {
           <Route path='/test'>
             <RequestGroupDonorView requestGroupId="603d9b41eb57fc06447b8a23" />
           </Route>
+          <Route path='/admin-group-view/:id'><AdminRequestGroupView /></Route>
```

Comment:
Can we change the url to `/request-group/:id` ?

Reply:
Done